install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -267,6 +267,7 @@ type ServerConfig struct {
 	DisableCompleteSnapshotJob        bool              `json:"disableCompleteSnapshotJob"`
 	InactivityPeriodForReposInDays    *int              `json:"inactivityPeriodForReposInDays"`
 	ShowSetupModal                    *bool             `json:"showSetupModal"`
+	isSingleOrgInstallation           *bool             `json:"isSingleOrgInstallation"`
```

Comment:
You also _must_ make this variable public (uppercase) else it won't get picked when marshalling from/to json